js/charts/clusteredhorizontalbar.js

```diff
@@ -1,6 +1,8 @@
-var COMMON = require('../extras/common.js')(),
-    UTIL = require('../extras/util.js')(),
-    LEGEND = require('../extras/legend_barcharts.js')();
+var d3 = require('d3');
+var COMMON = require('../extras/common.js')();
+var UTIL = require('../extras/util.js')();
+var LEGEND = require('../extras/legend_barcharts.js')();
```

Comment:
For now, we have used legend_barcharts but ultimately later we need a single common legend.js file. We can have it as it is for now.